### DISCUSSION TITLE
How to know what is the specific deps used by gradle for djl

### DISCUSSION THREAD

**Discussion by RESDXChgfore9hing** [deepjavalibrary/djl#2048] [Q&A] [answered]
I can compile and run the code,now i need to *export the codes with the dependencies into a neat zip/folder.
now how do i do this?do I use some gradle release plugin?or manual?

**Comment by frankfliu** [comment, marked as answer]
@RESDXChgfore9hing 

Here is the document about DJL's dependencies: https://docs.djl.ai/master/docs/development/dependency_management.html

If you want to package your project and distribute the project, you can take look this demo project: https://github.com/deepjavalibrary/djl-demo/tree/master/development/fatjar#how-to-distribute-djl-application

**Reply by RESDXChgfore9hing** [reply]
awesome will try it out,
